src/components/MoviesList/MoviesList.tsx

```diff
@@ -4,12 +4,38 @@ import { MovieCard } from '../MovieCard';
 
 interface Props {
   movies: Movie[];
+  query: string;
 }
 
-export const MoviesList: React.FC<Props> = ({ movies }) => (
-  <div className="movies">
-    {movies.map(movie => (
-      <MovieCard key={movie.imdbId} movie={movie} />
-    ))}
-  </div>
-);
+interface State {
+  querySearch: Movie[]
+}
+
+export class MoviesList extends React.Component<Props, State> {
+  state = {
+    querySearch: this.props.movies,
```

Comment:
```suggestion
    movies: this.props.movies,
```
It just movies, query don't influence it